packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -113,6 +120,14 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
             this.updateXY();
         }
         if (this.lineChartRef.current) {
+            if (this.preventDefaultHandler === undefined) {
+                this.preventDefaultHandler = (event: WheelEvent) => {
+                    if (event.ctrlKey) {
+                        event.preventDefault();
+                    }
+                };
+                this.chartRef.current.addEventListener('wheel', this.preventDefaultHandler);
+            }
```

Comment:
Could the listener be added in componentDidMount instead?

Reply:
I tried it but it didn't work. the view didn't want to display.